stable/prometheus-blackbox-exporter/values.yaml

```diff
@@ -100,11 +100,14 @@ serviceMonitor:
   ## https://github.com/coreos/prometheus-operator
   ##
   enabled: false
-  labels: {}
-  interval: 30s
-  scrapeTimeout: 30s
-  module: http_2xx
-  # The URL that blackbox will scrape
-  url: http://example.com/healthz
-  # Optional human readable URL that will appear in Prometheus / AlertManager
-  urlHumanReadable:  # www.changemeoriwillfail.com
+
+  # Default values that will be used for all ServiceMonitors created by `targets`
+  defaults:
+    labels: {}
+    interval: 30s
+    scrapeTimeout: 30s
+    module: http_2xx
+
+  targets:
+#    - name: example # Human readable URL that will appear in Prometheus / AlertManager
+#      url: http://example.com/healthz # The URL that blackbox will scrape
```

Comment:
Maybe you should put an example of optional values (labels, etc).

Reply:
Good point, updated the file.